### PR TITLE
Edit snapshot doc to updated support of hotplugged disks

### PR DESCRIPTION
**Pull request by ShellyKa13**
This is relevant since kubevirt release v0.47

### CODE REVIEW

File: docs/operations/snapshot_restore_api.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ When snapshotting a running vm the controller will check for qemu guest agent in
 
 There will be an indication in the vmSnapshot status if the snapshot was taken online and with or without guest agent participation.
 
-> *Note* Currently online vm snapshot is not supported with hotplugged disks, in this case the vm has to be turned off in order to take the snapshot (or all hotplugged disks unplugged).
+> *Note* Online snapshot with hotplugged disks is supported, only persistent hotplugged disks will be included in the snapshot.
```
Comment (mazzystr) [inline docs/operations/snapshot_restore_api.md:38]: sed -e 's/, / but /'

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Comment by mazzystr** [comment]
> kubevirt release v4.10

KubeVirt has not reached v1.0 yet.  Are you referring to a vendor release number?  If so please adjust your description.

File: docs/operations/snapshot_restore_api.md (hunk 1 above)
Comment (ShellyKa13) [reply docs/operations/snapshot_restore_api.md:38]: I dont think that 'but' fits here cause it doesn't contradict the fact that hotplugged disks is supported.. maybe more like 'pay attention' or 'bear in mind'.. what do you think?

**Review by stu-gott — COMMENTED** [review]
/approve

**Pull request merged**